contracts/Cross-chain/OmnichainGovernanceExecutor.sol

```diff
@@ -124,10 +126,13 @@ contract OmnichainGovernanceExecutor is ReentrancyGuard, BaseOmnichainController
             "OmnichainGovernanceExecutor::initialize:number of timelocks _should match the number of governance routes"
         );
         for (uint8 i; i < uint8(ProposalType.CRITICAL) + 1; ++i) {
-            require(
-                address(timelocks_[i]) != address(0),
-                "OmnichainGovernanceExecutor::initialize:invalid timelock address"
-            );
+            ensureNonzeroAddress(address(timelocks_[i]));
+            if (i > 0) {
+                require(
+                    timelocks_[i] != timelocks_[i - 1],
+                    "OmnichainGovernanceExecutor::initialize:duplicate timelock"
+                );
+            }
```

Comment:
I think this check wouldn't avoid to set these timelocks:
```
["0xABC", "0xDEF", "0xABC"]
```
The last timelock will be different from the previous one, but equal to the first one. I think I wouldn't do any checks for this. It's a privilege function, we can check the duplications offchain and we could re-execute it to fix the situation

Reply:
https://github.com/VenusProtocol/governance-contracts/pull/59/commits/2c94bd93f3559aa50688abcad257dac9d5f9c645